python/cudf/cudf/_lib/stream_compaction.pyx

```diff
@@ -144,13 +151,41 @@ def drop_duplicates(columns: list,
         if nulls_are_equal
         else null_equality.UNEQUAL
     )
+
+    cdef vector[order] column_order
+    column_order.reserve(cpp_keys.size())
+    cdef vector[null_order] null_precedence
+    null_precedence.reserve(cpp_keys.size())
+
+    for _ in range(cpp_keys.size()):
+        column_order.push_back(order.ASCENDING)
+        null_precedence.push_back(null_order.BEFORE)
```

Comment:
Can this use the other `vector` constructor with a default value?

https://github.com/cython/cython/blob/756b795532151f0a8bda7960fd91f4c663385ecf/Cython/Includes/libcpp/vector.pxd#L45

```suggestion
    cdef vector[order] column_order(cpp_keys.size(), order.ASCENDING)
    cdef vector[null_order] null_precedence(cpp_keys.size(), null_order.BEFORE)
```